tests/UI/campaigns/functional/BO/13_shopParameters/07_search/01_search/01_CRUDSearch.js

```diff
@@ -0,0 +1,153 @@
+require('module-alias/register');
+
+// Import utils
+const helper = require('@utils/helpers');
+
+// Import common tests
+const loginCommon = require('@commonTests/loginBO');
+
+// Import pages
+const dashboardPage = require('@pages/BO/dashboard');
+const searchPage = require('@pages/BO/shopParameters/search');
+const addSearchPage = require('@pages/BO/shopParameters/search/add');
+
+// Import data
+const SearchFaker = require('@data/faker/search');
+
+// Import test context
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_shopParameters_search_search_CRUDSearch';
+
+// Import expect from chai
+const {expect} = require('chai');
+
+// Browser and tab
+let browserContext;
+let page;
+let numberOfSearch = 0;
+
+const createAliasData = new SearchFaker();
+const editSearchData = new SearchFaker({alias: createAliasData.alias});
```

Comment:
Any reason why we don't update the alias ?

Reply:
When we update the alias, a new alias is created.
I don't know, maybe it's an expected behavior, we should verify it

Reply:
this could be an issue

Reply:
yes, it's a bug, I will create an issue :+1: